exercises/practice/atbash-cipher/.meta/example.cairo

```diff
@@ -0,0 +1,78 @@
+pub fn encode(phrase: ByteArray) -> ByteArray {
+    let mut result = "";
+
+    let mut i = 0;
+    let mut chunk_size = 0;
+    while i < phrase.len() {
+        let chr = phrase[i];
+
+        if is_whitespace(chr) || is_punctuation(chr) {
```

Comment:
```suggestion
        if !is_letter(chr) {
```
nit: we can ignore _all_ non-letter chars. Also makes the `is_punctuation` fn redundant.